### PR TITLE
feat(wallet): add fee granter to transaction signer

### PR DESCRIPTION
**Pull request by ygrishajev**
[DEMO](https://drive.google.com/file/d/1g5yqw0scRgHEbblJtt5PmW6N2otAlBkJ/view?usp=share_link)

### CODE REVIEW

**Comment by ygrishajev** [comment]
[UPDATED DEMO](https://drive.google.com/file/d/1ODtSBppFw96QPpekdf5HxvjpOsbHdnPb/view?usp=share_link)

File: apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx [hunk 1]
```diff
@@ -4,29 +4,33 @@ import { FormattedTime } from "react-intl";
...
-  allowance: AllowanceType;
+  allowance: AllowanceType | { granter: "Connected Wallet" };
   children?: ReactNode;
+  onSelect?: () => void;
+  selected?: boolean;
 };
 
-export const AllowanceGrantedRow: React.FunctionComponent<Props> = ({ allowance }) => {
+export const AllowanceGrantedRow: React.FunctionComponent<Props> = ({ allowance, selected, onSelect }) => {
+  const isFull = "allowance" in allowance;
```
Comment (baktun14) [inline apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx]: How does that work? If there's an allowance, it's full?

**Review by baktun14 — APPROVED** [review]

File: apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx (hunk 1 above)
Comment (ygrishajev) [reply apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx]: after u asked this question decided to do it a bit differently. Hopefully simpler



https://github.com/akash-network/cloudmos/pull/238/files#diff-3a051be7611032924e7ddd61b9c21e5b15bfdfd97fef2f5e76f614c825ef6758R275

Comment (baktun14) [reply apps/deploy-web/src/components/authorizations/AllowanceGrantedRow.tsx]: Ah clearer this way! 👍

**Pull request merged**